crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/MaatCourtDataService.java

```diff
@@ -97,6 +100,7 @@ public IOJAppealDTO getIOJAppealFromRepId(Integer repId, String laaTransactionId
                 .body(BodyInserters.fromValue(postBody))
                 .retrieve()
                 .bodyToMono(responseClass)
+                .onErrorResume(WebClientResponseException.NotFound.class, notFound -> Mono.empty())
```

Comment:
Hi Mike
There is no 404 NOT FOUND returned on createAssessment. It's required only for get/find urls